builtInFunctions/blockDataHandler.go

```diff
@@ -0,0 +1,39 @@
+package builtInFunctions
+
+import (
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	vmcommon "github.com/multiversx/mx-chain-vm-common-go"
+)
+
+type blockDataHandler struct {
```

Comment:
please rename this, we have this struct that has a set method called almost the same `SetBlockDataHandler`. It creates confusion.